File: 002.c
#include <stdio.h>

void main() {
  const int LIMIT = 4000000;
  int lowerTerm = 1;
  int higherTerm = 2;
  int total = 0;
  int temp;

  do {
    if(higherTerm % 2 == 0) {
      total += higherTerm;
    }
    temp = higherTerm;
    higherTerm = lowerTerm + higherTerm;
    lowerTerm = temp;
  } while(higherTerm <= LIMIT);

  printf("%d\n", total);

}

File: source/002.py
#!/usr/bin/python

lowerTerm = 1
higherTerm = 2
total = 0

while higherTerm < 4000000:
  if higherTerm % 2 == 0:
    total = total + higherTerm
  temp = higherTerm
  higherTerm = higherTerm + lowerTerm
  lowerTerm = temp

print total
